gateway/api/management/commands/update_jobs_statuses.py

```diff
@@ -57,6 +62,15 @@ def handle(self, *args, **options):
                 if job_handler:
                     logs = job_handler.logs(job.ray_job_id)
                     job.logs = check_logs(logs, job)
+                # check if job is resource constrained
```

Comment:
One question here @psschwei that it was not clear for me. I see that we are using `logs` , do we wanted to have this logic inside the `if`? Right now it's outside so probably this `logs` will be always None.

Reply:
I don't know if we need to move it inside the if, but the check should probably be against `job.logs` instead of the temp variable, i.e. `if no_resources_log in job.logs:` wdyt?

Reply:
Reviewing the code I think it has sense to move it under `job_handler`. Basically that's the point where we know that we have the cluster running. Before it I don't think it has sense to check it, probably. And yes, I agree. We should check `job.logs` 💯